cannon/mipsevm/singlethreaded/state.go

```diff
@@ -0,0 +1,329 @@
+package singlethreaded
```

Comment:
This file was removed as the singlethreaded VM is no longer supported. Could you please rebase this PR on the latest develop?